core/src/main/scala/org/ensime/util/FileUtil.scala

```diff
@@ -84,14 +84,14 @@ object FileUtils {
     }
   }
 
-  def writeDiffChanges(changes: List[FileEdit], cs: Charset): Either[Exception, File] = {
+  def writeDiffChanges(changes: List[FileEdit])(implicit cs: Charset): Either[Exception, File] = {
     //TODO: add support for NewFile and DeleteFile
```

Comment:
it's ok to add it as a separate commit here. Since it is implicit, can you perhaps call it implicitly instead of explicitly?